backend/app/cmd/server.go

```diff
@@ -98,7 +98,7 @@ type ServerCommand struct {
 			SMTPUserName string        `long:"user" env:"USER" description:"[deprecated, use --smtp.username] enable TLS"`
 			TLS          bool          `long:"tls" env:"TLS" description:"[deprecated, use --smtp.tls] SMTP TCP connection timeout"`
 			TimeOut      time.Duration `long:"timeout" env:"TIMEOUT" default:"10s" description:"[deprecated, use --smtp.timeout] SMTP TCP connection timeout"`
-			MsgTemplate  string        `long:"template" env:"TEMPLATE" description:"message template file"`
+			MsgTemplate  string        `long:"template" env:"TEMPLATE" description:"message template file" default:"email_verification.html.tmpl"`
```

Comment:
This should be reflected in Readme

Reply:
Actually, if we have 100% other template filenames hardcoded, I would suggest to instead remove this file from configuration and document it alongside with others.